services/sputnik/SputnikNearService/services/SputnikDaoService.ts

```diff
@@ -143,22 +143,23 @@ export class SputnikDaoService {
         id,
         deadline,
       },
-      gas: GAS_VALUE,
+      gas: gas ? new BN(Number(gas) * 10 ** 15) : GAS_VALUE,
```

Comment:
I see that 10 ** 15 is being used in multiple places. Is there sense to move it to constant?